solution/ui/regulations/js/src/components/tooltips/CopyCitation.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div class="action-btns">
+        <ActionBtn
+            :selected-action="selectedAction"
+            :status="copyStatus"
+            action-type="link"
+            @action-btn-click="handleActionClick"
+        ></ActionBtn>
+        <ActionBtn
+            :selected-action="selectedAction"
+            :status="copyStatus"
+            action-type="citation"
+            @action-btn-click="handleActionClick"
+        ></ActionBtn>
+    </div>
+</template>
+
+<script>
+import ActionBtn from "./ActionBtn.vue";
+
+export default {
+    name: "CopyCitation",
+
+    components: {
+        ActionBtn,
+    },
+
+    props: {
+        formattedCitation: {
+            type: String,
+            required: true,
+        },
+        hash: {
+            type: String,
+            required: true,
+        },
+    },
+
+    data() {
+        return {
+            selectedAction: null,
+            copyStatus: "idle",
+        };
+    },
+
+    watch: {
+        async copyStatus (newStatus, oldStatus) {
+            if (
+                newStatus === "pending" &&
+                (oldStatus === "idle" || oldStatus === "success")
```

Comment:
Just trying to understand the logic. There are three statuses from what I can gather.  Is it possible for an old and new status to both be "pending" at the same time or is there another stautus from these three.  Just because if not then wouldnt if newsttus ==="pending" have the same affect.